### PR TITLE
[SP-4632] Backport of PDI-17467 - Kitchen, Pan, Carte, Import, PurgeU…

### PR DESCRIPTION
**Pull request by pentaho-lmartins**
…tility require plain-text password (7.1 Suite)

Same as #5783 (cherry pick of #5653, #5720 and #5757) to 7.1.

@cravobranco @ricardosilva88 @ssamora

### CODE REVIEW

**Comment by ricardosilva88** [comment]
This wigman output seems to be a bit odd?

**Comment by pentaho-lmartins** [comment]
Updated PR.

**Comment by pentaho-lmartins** [comment]
Updated PR

File: plugins/pdi-pur-plugin/test/com/pentaho/di/purge/RepositoryCleanupUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.reflect.Whitebox.getInternalState;
+import static org.powermock.reflect.Whitebox.setInternalState;
+
+@RunWith( PowerMockRunner.class )
+@PrepareForTest( Client.class )
+public class RepositoryCleanupUtilTest {
+
```
Comment (ricardosilva88) [inline plugins/pdi-pur-plugin/test/com/pentaho/di/purge/RepositoryCleanupUtilTest.java]: @Before
  public void setup() throws Exception {
    KettleClientEnvironment.init();
  }

File: engine/test-src/org/pentaho/di/www/CarteTest.java [hunk 2]
```diff
@@ -21,22 +21,39 @@
...
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.reflect.Whitebox.getInternalState;
 
 /**
  * Created by ccaspanello on 5/31/2016.
  */
+@RunWith( PowerMockRunner.class )
+@PrepareForTest( Client.class )
 public class CarteTest {
```
[the remark this answers is not in the document]
Comment (pentaho-lmartins) [reply engine/test-src/org/pentaho/di/www/CarteTest.java]: Nope, that was removed in #5757.

File: plugins/pdi-pur-plugin/test/com/pentaho/di/purge/RepositoryCleanupUtilTest.java (hunk 1 above)
Comment (pentaho-lmartins) [reply plugins/pdi-pur-plugin/test/com/pentaho/di/purge/RepositoryCleanupUtilTest.java]: Nope, that was removed in #5757.

**Comment by pentaho-lmartins** [comment]
@ricardosilva88, please review and merge.

**Pull request merged**